### ISSUE TITLE
False positive when `__foo` is used for positional-only argument

### ISSUE THREAD

**Issue by rsokl** [microsoft/pyright#3186]
Using pyright 1.1.299.

```python
# pyright: strict

def f(p: int, /, x: int, y: str):  # OK
    ...

def g(__p: int, x: int, y: str):  # error: Type annotation is missing for parameter "x" (reportMissingParameterType)
    ...

```

**Comment by erictraut** [comment]
Ooh, good find. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.230, which I just published. It will also be included in the next release of pylance.
